src/bloom.cpp

```diff
@@ -62,6 +90,19 @@ CBloomFilter::CBloomFilter(unsigned int nElements,
     setup(nElements, nFPRate, nTweakIn, nFlagsIn, true, nMaxFilterSize);
 }
 
+CBloomFilter::CBloomFilter(unsigned int nElements,
+    double nFPRate,
+    unsigned int nTweakIn,
+    unsigned char nFlagsIn,
+    bool guaranteeFPR,
```

Comment:
-guaranteeFPR
+fGuaranteeFPR

it's a small distinction but we try to maintain a naming convention...  f for flags/bools, n for numbers, v for vectors etc...  it helps when reading the code to know what the value is pertaining to without having to look it up each time.   I know there are some old cases, even in bloom.cpp that should be changed but at least the new stuff we try to maintain some kind of naming consistency.

Reply:
Done.